codeHF/config_input.sh

```diff
@@ -77,4 +77,8 @@ case $INPUT_CASE in
     JSON="$JSONRUN5"
     ISINPUTO2=1
     ISMC=1;;
+  12)
+    INPUT_LABEL="p-p MC 13 TeV Xic, LHC19g6f"
+    INPUT_DIR="/data/Xi_c_MC/LHC19g6f3/264347"
+    ISMC=1;;
```

Comment:
I tried to run this and it fails. Please fix it.

Reply:
I readapted the label name, let me know if it is ok.
About the code you ping: actually I have not run it in the aliceml machine, but I did all my tests locally in my laptop, downloading some MC file from the GRID. I checked the path in the machine and it is correct, but I think you see the failure because the files `Kinematics.root` and `galice.root` are not present.

Reply:
Yes, I thought you are the one who downloaded the data there.
Who was it then?

Reply:
No I have never touched anything in aliceml. I think @zhangbiao-phy took care of it.

Reply:
> No I have never touched anything in aliceml. I think @zhangbiao-phy took care of it.

Hi @mfaggin, I think you can test it now. I have downloaded both of them

Reply:
Thanks @zhangbiao-phy